app/src/main/kotlin/com/mapbox/vision/teaser/replayer/ArReplayNavigationActivity.kt

```diff
@@ -376,45 +403,16 @@ class ArReplayNavigationActivity : AppCompatActivity(), MapboxMap.OnMapClickList
         )
     }
 
-    private fun DirectionsRoute.getRoutePoints(): Array<RoutePoint> {
-        val routePoints = arrayListOf<RoutePoint>()
-        legs()?.forEach { leg ->
-            leg.steps()?.forEach { step ->
-                val maneuverPoint = RoutePoint(
-                    GeoCoordinate(
-                        latitude = step.maneuver().location().latitude(),
-                        longitude = step.maneuver().location().longitude()
-                    ),
-                    step.maneuver().type().mapToManeuverType()
-                )
-                routePoints.add(maneuverPoint)
-
-                step.geometry()
-                    ?.buildStepPointsFromGeometry()
-                    ?.map { geometryStep ->
-                        RoutePoint(
-                            GeoCoordinate(
-                                latitude = geometryStep.latitude(),
-                                longitude = geometryStep.longitude()
-                            )
-                        )
-                    }
-                    ?.let { stepPoints ->
-                        routePoints.addAll(stepPoints)
-                    }
-            }
-        }
-        return routePoints.toTypedArray()
-    }
-
-    private fun hideMapView() {
+    private fun hideTools() {
+        playback_seek_bar_view.hide()
         map_container.removeView(mapView)
-        mapVisible = false
+        toolsVisible = false
```

Comment:
for me better to move this flag outside
Hide and show tools is about UI components

set `toolsVisible != toolsVisible` or `toolsVisible = toolsVisible.not` when `ar_view` clicked

Reply:
good idea, fixed